src/components/error-summary/error-summary.yaml

```diff
@@ -0,0 +1,28 @@
+variants:
+  - name: default
+    data:
+      title: Message to alert the user to a problem goes here
+      description: Optional description of the errors and how to correct them
+      listOfErrors:
+        items:
+        -
+          content: Descriptive link to the question with an error
+          href: '#example-error-1'
+        -
+          content: Descriptive link to the question with an error
+          href: '#example-error-1'
+  - name: error-list-is-bulleted
```

Comment:
We don't have any examples in Elements that use the bulleted list. Unless there's a good reason to encourage users to do this, I don't think we should have an example for it.

Reply:
i guess then it doesnt need a type after all ...